pkg/kritis/metadata/containeranalysis/cache_test.go

```diff
@@ -139,12 +141,10 @@ func TestNCache(t *testing.T) {
 	}
 }
 
-func makeRawAttestationPgp(signature, id string) metadata.RawAttestation {
-	return metadata.RawAttestation{
-		SignatureType: metadata.PgpSignatureType,
-		Signature: metadata.RawSignature{
-			Signature:   signature,
-			PublicKeyId: id,
-		},
+func makePgpAttestation(signature, id string) cryptolib.Attestation {
```

Comment:
Nit: you didn't really add this function, but generally in tests objects like this one are constructed directly instead of using a helper function. Helper functions are only really used if the object is very complicated and only a few parameters change test to test.

Reply:
Done, removed